src/components/InterestForm/index.tsx

```diff
@@ -86,46 +125,51 @@ export default function InterestForm({ caseData }: { caseData: CaseListing }) {
     <FormContainer>
       <H3>Submit Interest</H3>
       {submitted ? (
-        <P>We have received your submission!</P>
+        <P>Your submission has been received!</P>
       ) : (
-        <>
-          <FormQuestion $color={COLORS.greyDark}>
-            What role(s) are you applying for?
-          </FormQuestion>
-          <RadioGroup>
-            {radioOptions.map(option => (
-              <Radio key={option}>
-                <RadioInput
-                  id={`radio${option}`}
-                  type="radio"
-                  name="radioOptions"
-                  value={option}
-                  checked={rolesInterested === option}
-                  onChange={event => setRolesInterested(event.target.value)}
-                />
-                <RadioLabel htmlFor={`radio${option}`}>{option}</RadioLabel>
-              </Radio>
-            ))}
-            {missingInfo && rolesInterested === '' && (
-              <ErrorText>Must select your preferred role</ErrorText>
+        <Flex $gap="30px" $direction="column">
+          {(listingData.listing_type === 'CASE' ||
+            listingData.listing_type === 'INT') && (
+            <DateInput
+              label="What is the earliest date you can contact the client?"
+              required
+              error={getErrorText()}
+              name="startDate"
+              value={startDate}
+              setValue={setStartDate}
+            />
+          )}
+          {listingData.listing_type === 'CASE' && !interpretation && (
+            <RadioGroup
+              name="reason"
+              required
+              value={needsInterpreter}
+              setValue={setNeedsInterpreter}
+              options={['Yes', 'No']}
+              label="Do you need language interpretation help for the client?"
```

Comment:
Design question, but what if we rephrased this question to something like:

"Will you need language interpretation assistance to represent this client?"

(cc @kyrenetam @varortz)

Reply:
yeah let's go ahead and update it

Reply:
sg!

Reply:
<img width="525" alt="Screen Shot 2024-04-17 at 2 00 54 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/64336644/43fc0dff-2f4b-45fc-b4a5-5312dfa116e7">
This is how it looks now. the question is a bit long though. until we have a better description, i'll keep it like this

Reply:
Yeah, I think it's ok if it looks a bit long, but let's check in with @kyrenetam  on this!